lib/kernel/host/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ endif()
 if(X86_64 OR I386)
   if(LLVM_3_6)
     message(STATUS "OpenCL 2.0 atomics only works with LLVM >= 3.7")
+  elseif(LLVM_3_9)
```

Comment:
Can you use OLDER_THAN_LLVM_3_9 here also (in the else branch) to avoid this (likely) breaking in LLVM 4.0?